### PR TITLE
[Review] Request from 'aduffeck' @ 'SUSE/machinery/review_151126_adapt_packages_data_model'

### PR DESCRIPTION
**Pull request by aduffeck**
Please review the following changes:
  * 4802858 Fix hound issues
  * 7cf54df Migrate fixture system descriptions
  * 29e10f9 Adapt packages data model for v6 (to add support for dpkg packages)
  * 4326ecb Add unchanged schema files for v6
  * e6166ea Add migration for format version 6

### CODE REVIEW

File: html/partials/packages.html.haml [hunk 1]
```diff
@@ -23,7 +23,7 @@
                   %th.medium Vendor
                   %th Checksum
               %tbody
-                - packages.each do |package|
+                - packages.packages.each do |package|
```
Comment (mauromorales) [inline html/partials/packages.html.haml]: I know this is the code following the model but it just looks bad to me to have packages.packages, what do you think?

Comment (aduffeck) [inline html/partials/packages.html.haml]: I agree that it looks a bit weird. We have the same problem with `services.services` and in a less dramatic form with `config_files.files` etc.

Unfortunately I don't know of a good way to solve that right now...

**Comment by mauromorales** [comment]
lgtm

File: html/partials/packages.html.haml (hunk 1 above)
Comment (mauromorales) [inline html/partials/packages.html.haml]: got it, ok let's look at it later on

**Comment by cornelius** [comment]
Shouldn't this go to the `ubuntu_inspection` branch instead of `master`?

**Comment by aduffeck** [comment]
Yes, definitly. I just checked my history and I told git review to do so, but apparently that's not the proper way to do it:

`TARGET_BRANCH=ubuntu_inspection git review create`

I'll create a new pull request to the right branch once manual fixed the problem with the integration tests.

**Pull request closed without merging**